### PR TITLE
running parallel regression tests

### PR DESCRIPTION
**Pull request by tskille**
force mpi-run to use same time stepping as serial run

### CODE REVIEW

File: tests/run-regressionTest.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-# This runs a simulator, then compares the summary, restart and init
-# files against a reference.
+# This performs a serial and a parallel for a simulator,
+# then compares the summary and restart files from the two runs.
+# Meant to track regression in parallel simulators.
```
Comment (akva2) [inline tests/run-regressionTest.sh:5]: why are you changing the regression test script to the parallel one?

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by tskille** [comment]
This was not successful. All test did go through locally but this was not the case on Jenkins.  I'm closing this.

**Pull request closed without merging**